src/dotnet-new3/Program.cs

```diff
@@ -24,8 +23,9 @@ public static Task<int> Main(string[] args)
                 }
             }
 
-            Command newCommand = New3CommandFactory.Create();
-            return ParserFactory.CreateParser(newCommand).Parse(args).InvokeAsync();
+            RootCommand rootCommand = new RootCommand();
```

Comment:
was this the core bug in the way the parser was created that you mentioned to me? the one that prevented the tests from running the same way against the SDK repo as they do here?

Reply:
no, the bug was here: https://github.com/dotnet/templating/pull/4262#discussion_r792441921

However, this is also difference from SDK.